test/jdk/sun/security/pkcs12/KeytoolOpensslInteropTest.java

```diff
@@ -67,18 +67,19 @@
 public class KeytoolOpensslInteropTest {
 
     public static void main(String[] args) throws Throwable {
-        String opensslPath = OpensslArtifactFetcher.getOpenssl1dot1dotStar();
+        String opensslPath = OpensslArtifactFetcher.getOpensslPath();
         if (opensslPath != null) {
-            // if preferred version of openssl is available perform all
+            // if the current version of openssl is available, perform all
             // keytool <-> openssl interop tests
             generateInitialKeystores(opensslPath);
             testWithJavaCommands();
             testWithOpensslCommands(opensslPath);
         } else {
-            // since preferred version of openssl is not available skip all
+            // since the current version of openssl is not available, skip all
```

Comment:
If this test is to check interoperability with OpenSSL, shouldn't we throw a skippedexception or an error instead of just running with java?

Reply:
If Openssl is not available in the instance, this test is using preexisting PKCS12 files generated with an older Openssl version for testing compatibility between openssl and keytool, these files are located in the ./params directory.
As discussed with Matthew, I will separate both paths adding an extra @test tag, one test will generate the PKCS12 on the fly and throw an SkippedException if Openssl is not available, and the other will use the preexisting files for the verifications.